assets/queries/terraform/kubernetes/tiller_is_deployed/query.rego

```diff
@@ -0,0 +1,120 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].resource[resourceType]
+
+	checkMetadata(resource[name].metadata)
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("%s[%s].metadata", [resourceType, name]),
```

Comment:
you could be more specific on the search key on where in the metadata the attribute is wrong, maybe even have *checkMetadata()* return the path as a string.

Reply:
Done 👍